snap/snapcraft.yaml

```diff
@@ -48,15 +56,16 @@ parts:
       - freeglut3
       - libglu1
       - libpulse0
+      - libslang2
       # Are all the below needed?
       - libxcb-xinerama0
       - libxinerama1
       - libgtk2.0-dev
       - libpulse-dev
       - libgl1-mesa-dev
```

Comment:
It's strange that you include many `-dev` libraries in stage-packages. `-dev` libraries provide source and header files used to compile software. `libpulse-dev` contains the source code used by applications which connect to the `libpulse` library.

`stage-packages` are added to the resulting snap so the only reason to include these libraries is if your application itself contains a compiler and will compile code during runtime.

If these libraries are only needed for compiling your application, you should add them to `build-packages` instead.